apps/meteor/app/livechat/server/api/v1/room.ts

```diff
@@ -350,6 +320,54 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
```

Comment:
I recommend you declare the endpoint using our new pattern instead of the deprecated declaration pattern.

 i.e: `API.v1.post()`